ui/plugin-lifecycle/vcd-ui/common/index.d.ts

```diff
@@ -0,0 +1,89 @@
+/**
```

Comment:
vcd-ui/ folder should be removed from project.  All necessary definitions are exported by @vcd/sdk

Reply:
It is removed in last commit.